modules/serverless-https-redirect-service.adoc

```diff
@@ -6,7 +6,8 @@
 [id="serverless-https-redirect-service_{context}"]
 = HTTPS redirection per service
 
-You can enable or disable HTTPS redirection for a service by configuring the `networking.knative.dev/httpOption` annotation, as shown in the following example:
+// need better details from eng team about use case to update this topic
+You can enable or disable HTTPS redirection for a service by configuring the `networking.knative.dev/httpOption` annotation. The following example shows how you can use this annotation in a Knative `Service` YAML object:
```

Comment:
Since serving 1.1 (serverless 1.22) we introduced `-` style annotation `networking.knative.dev/http-protocol`. 
It is still possible to use both `networking.knative.dev/httpOption` and `networking.knative.dev/http-protocol` so we update the docs later?